config/src/tokens.ts

```diff
@@ -216,6 +216,14 @@ export const tokenList: TokenInfo[] = [
     coingeckoId: 'golem',
     sinceBlock: 11281056,
   },
+  {
+    name: 'Habitat Token',
+    symbol: 'HBT',
+    address: '0x0aCe32f6E87Ac1457A5385f8eb0208F37263B415',
+    decimals: 10,
+    coingeckoId: 'habitat',
```

Comment:
The new code on the master branch no longer uses coingecko so this is now not needed.